k8s/oidc-vault/README.md

```diff
@@ -0,0 +1,435 @@
+# Using SPIRE and OIDC to authenticate workloads to retrieve Vault secrets
+
+This tutorial builds on the [Kubernetes Quickstart](https://spiffe.io/spire/try/getting-started-k8s/) guide to describe how a SPIRE identified workload can, using a JWT-SVID, authenticate to an HashiCorp Vault server, assume an identity, and retrieve secrets. This avoids the need to spread Vault credentials.
```

Comment:
how about:
`This tutorial builds on the "Kubernetes Quickstart" guide to describe how to set up OIDC Federation between a SPIRE Server and a Vault Server. This will allow a SPIRE-identified workload to retrieve secrets from the Vault Server by presenting no more than its JWT-SVID. Using this technique the workload won't need to authenticate itself against the Vault Server using auth methods like AppRole or Username & Password.`

Reply:
Better! That clarifies it. I have a few tweaks. The official Vault docs don't seem to capitalize server in "Vault server". 
```
This tutorial builds on the "Kubernetes Quickstart" guide to describe how to set up OIDC Federation between a SPIRE Server and a Vault server. This will allow a SPIRE-identified workload to retrieve secrets from the Vault server by presenting no more than its JWT-SVID. Using this technique the workload won't need to authenticate itself against the Vault server using authentication methods like AppRole or Username & Password.
```

Reply:
got it, I recently pushed a commit capitalizing every single "Vault Server" in this document, I'll revert these changes.

Reply:
Bummer, sorry about that

Reply:
Tweaked this a little bit to make it clear that you need to authenticate using the JWT-SVID:
```
This tutorial builds on the [Kubernetes Quickstart](https://spiffe.io/spire/try/getting-started-k8s/) guide to describe how to set up OIDC Federation between a SPIRE Server and a Vault server. This will allow a SPIRE-identified workload to authenticate against a federated Vault server by presenting no more than its JWT-SVID. Using this technique the workload won't need to authenticate itself against the Vault server using another authentication method like AppRole or Username & Password.
```